src/plugins/discover/public/application/main/components/layout/discover_main_content.tsx

```diff
@@ -128,15 +133,16 @@ export const DiscoverMainContent = ({
           {showChart && isChartAvailable && <EuiHorizontalRule margin="none" />}
           {viewMode === VIEW_MODE.DOCUMENT_LEVEL ? (
             <DiscoverDocuments
-              viewModeToggle={viewModeToggle}
+              viewModeToggle={viewModeToggle()}
```

Comment:
For clarity I think we should rename it to something like viewModeToggle={renderViewModeToggle()}

Reply:
Updated in https://github.com/elastic/kibana/pull/178916/commits/186f78b53ebc6cf2166bb73a2daffa6f267ef003

Reply:
> super duper nit
Nah, this needed changing. It was being passed about differently, for the pattern analysis component it was being passed I as a reference, but for the original tabs it was being called and the result passed in.
Definitely needed renaming.